session/bootstrap.go

```diff
@@ -549,6 +550,19 @@ const (
     	key(parent_table_id, create_time),
     	key(create_time)
 	);`
+
+	// CreateGlobalTask is a table about global task.
+	CreateGlobalTask = `CREATE TABLE IF NOT EXISTS mysql.tidb_global_task (
+		id BIGINT(20) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+		type VARCHAR(256) NOT NULL,
+		dispatcher_id VARCHAR(256),
+		state VARCHAR(64) NOT NULL,
+		start_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+		end_time TIMESTAMP,
+		meta LONGBLOB,
+		concurrency INT(11),
+		step INT(11)
+	);`
```

Comment:
Do we need add some index for this table?

Reply:
@wjhuang2016 Please address this comment.